frameworks/digital-outcomes-and-specialists/questions/briefs/questionAndAnswerSessionDetailsSpecialists.yml

```diff
@@ -0,0 +1,46 @@
+id: questionAndAnswerSessionDetails
+name: Question and answer session details
+question: Question and answer session details
+question_advice: |
+  <br>
```

Comment:
Do we want this `<br>` tag in here? Not sure if intentional or not? Had a search in the repo and looks like we use this very infrequently so wondered if there is a different way to do this or not...